src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+  const subtotal = basePrice * days;
+
+  if (days < shortTerm) {
+    return subtotal;
+  } else if (days >= shortTerm && days < longTerm) {
+    return subtotal - shortTermDiscount;
+  } else if (days >= longTerm) {
+    return subtotal - longTermDiscount;
+  }
```

Comment:
Here you can simplify your code to be more maintanable and readable if you use something like

```js

if (days < shortTerm) {
    return subtotal;
}

if (days >= shortTerm && days < longTerm) {
    return subtotal - shortTermDiscount;
}

return subtotal - longTermDiscount;

```

If your code enter in one of the ifs, the return of the if will be returned. Else, will be returned the default return, that is the subtotal - longTermDiscount. Make sense?

But your code is great! Good job! 🎉